### PR TITLE
added steps for comps.xml addition into a repo

### PR DESCRIPTION
**Pull request by kkaarreell**
Adds support for package group definitions and comps.xml addition into an existing repo.

### CODE REVIEW

**Comment by kkaarreell** [comment]
I will fixed most of the findings tomorrow.

File: dnf-docker-test/features/steps/groups_steps.py [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+                 |          | default    | bar   |
+
+         Scenario: Installing package group from background
+              When I enable repository "base"
+              Then I successfully run "dnf -y group install minimal"
+    """
+    HEADINGS_GROUP = ['Group', 'Tag', 'Value']
+    GROUP_TAGS_REPEATING = ['mandatory', 'default', 'optional']
+    GROUP_TAGS = ['is_default', 'is_uservisible', 'description'] + \
+                 GROUP_TAGS_REPEATING
```
Comment (kkaarreell) [inline dnf-docker-test/features/steps/groups_steps.py]: What is the preferred way of fixing this? Should I indent only using 4 spaces or should I rather use one long line?

File: dnf-docker-test/features/steps/groups_steps.py [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+    ========= ===== =======
+
+    *Tag* is describing characteristics of the respective
+    package group.Supported tags are:
+
+    ============== ===============
+         Tag        Default value 
+    ============== ===============
+    is_default          false     
+    is_uservisible      true
```
Comment (kkaarreell) [inline dnf-docker-test/features/steps/groups_steps.py]: Ignoring these as the text is a part of the documentation.

**Comment by mluscon** [comment]
Merged. Thank you.

**Comment by ignatenkobrain** [comment]
@mluscon, you know that there is `Merge` button, right? ;)

**Comment by mluscon** [comment]
yes, but I wanted to fix [E127] before merging.

**Pull request closed without merging**